woocommerce/woocommerce-ios
# [Mobile Payments] Payments menu design updates

Closes: #10867


## Description


As we work on the Payments menu for TTPUK, we're taking the opportunity to do a little tidying up.

This PR updates various strings, images, and sections in the Payments menu and Hub menu to match the updated designs.

### Designs
| Hub menu | Payments menu (before TTP) | Payments menu (after TTP) |
| --- | --- | --- |
![Hub menu design with updated description for Payments item highlighted](https://github.com/woocommerce/woocommerce-ios/assets/2472348/c614cb3f-56b6-4a88-9c2e-466e7db56ecd) | ![Hub menu design with new section for Pay in Person, new section header for Tap to Pay, and updated string for About Tap to Pay row all highlighted](https://github.com/woocommerce/woocommerce-ios/assets/2472348/22bb3691-840d-46d7-b275-07fa32c16efc) | ![Hub menu design after first Tap to Pay payment, with updated Share Feedback row title and icon highlighted](https://github.com/woocommerce/woocommerce-ios/assets/2472348/233653bb-c4a1-4852-ba19-7923ae0c1caf)


## Testing instructions


1. Delete the app from your device
2. Build and run from Xcode
3. Log in and select a US or UK based WooPayments store
4. Tap `Menu`
5. Observe that the Payments item's description has been updated and matches the designs above
6. Tap `Payments`
7. Observe that the Payments menu matches the designs above for before a TTP payment
8. Tap `Set Up Tap to Pay on iPhone` and go through the flow, including a test payment
9. Dismiss the order details
10. Observe that the payments menu matches the designs above for after a TTP payment (notably the change to the Share Feedback row)

N.B. Ignore the WooPayments Balances row, if you see it. It's feature flagged off in releases.

## Screenshots



https://github.com/woocommerce/woocommerce-ios/assets/2472348/1459159c-6ce4-4617-a847-29b2370b96d1


---
